package.json

```diff
@@ -1,11 +1,19 @@
 {
   "name": "magic-comments-loader",
-  "version": "1.4.2",
-  "description": "Add webpack magic comments to your dynamic imports during build time",
+  "version": "1.5.0",
+  "description": "Add webpack magic comments to your dynamic imports at build time",
   "main": "dist",
   "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/cjs/index.cjs",
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
   "engines": {
-    "node": ">=10.3.0"
+    "node": ">=12.17.0"
```

Comment:
Same reason as [this](https://github.com/morganney/magic-comments-loader/pull/18/files#r1237845200).